egs/chime4/s5_1ch/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -91,12 +91,65 @@ if [ ! -d exp/tri3b_tr05_multi_${train} ]; then
   exit 1;
 fi
 
-local/nnet3/run_ivector_common.sh \
-  --stage $stage --nj $nj \
-  --train-set "$train_set" --gmm $gmm \
-  --test-sets "$test_sets" \
-  --num-threads-ubm $num_threads_ubm \
-  --nnet3-affix "$nnet3_affix"
+if $decode_only; then
+  mdir=`pwd`
+  # check ivector extractor
+  if [ ! -d $mdir/exp/nnet3${nnet3_affix}/extractor ]; then
+    echo "error, set $mdir correctly"
+    exit 1;
+  elif [ ! -d exp/nnet3${nnet3_affix}/extractor ]; then
+    echo "copy $mdir/exp/nnet3${nnet3_affix}/extractor"
+    mkdir -p exp/nnet3${nnet3_affix}
+    cp -r $mdir/exp/nnet3${nnet3_affix}/extractor exp/nnet3${nnet3_affix}/
```

Comment:
I don't understand what is going on here.  mdir is pwd, so this is just copying something to itself.   It doesn't make sense to me.